src/rules/style/relational_symbols.rs

```diff
@@ -0,0 +1,87 @@
+use std::collections::BTreeMap;
+
+use crate::ast::FortitudeNode;
+use crate::settings::Settings;
+use crate::{ASTRule, Rule, Violation};
+use lazy_static::lazy_static;
+use tree_sitter::Node;
+
+lazy_static! {
+    static ref RELATIONAL_NAMES: BTreeMap<&'static str, &'static str> = {
+        BTreeMap::from([
+            (".gt.", ">"),
+            (".ge.", ">="),
+            (".lt.", "<"),
+            (".le.", "<="),
+            (".eq.", "=="),
+            (".ne.", "/="),
+        ])
+    };
+}
```

Comment:
```suggestion
fn map_relational_symbols(name: &str) -> Option<&'static str> {
    match name {
        ".gt." => Some(">"),
        ".ge." => Some(">="),
        ".lt." => Some("<"),
        ".le." => Some("<="),
        ".eq." => Some("=="),
        ".ne." => Some("/="),
        _ => None,
    }
}
```

Reply:
Ah nice, yes that's much more idiomatic! Although we know it will be exactly one of these, so can return just `&'static str` and use `_ => unreachable!()`